src/schema/schema.graphql

```diff
@@ -1347,6 +1347,21 @@ type AutoCompleteResponse {
 	match: [AutoCompleteMatch]
 }
 
+type AutoCompleteGALResponse {
+	contacts: [AutoCompleteCN]
```

Comment:
instead of AutoCompleteCN can we use Contact? as per my understanding AutoCompleteGal always returns contact objects

Reply:
ZimbraCalResType attribute is not available on type Contact. And this attribute is being used to distinguish users from resources.

Reply:
you can add it in Contact type, contacts can have any number of custom fields